lms/static/sass/search/_search.scss

```diff
@@ -0,0 +1,174 @@
+.search-bar,
+.course-index .search-bar,
+.dashboard .search-bar {
+
+  @include box-sizing(border-box);
+  position: relative;
+  padding: 5px;
+  font-family: $sans-serif;
+
+  form > div {
+    position: relative;
+  }
+
+  .search-field {
+    @include box-sizing(border-box);
+    @include border-radius(4px);
```

Comment:
Border-radius is now supported widely enough (excepting IE8, which is not a supported browser) that you can drop using the older bourbon border-radius mixin and instead use the border-radius css property directly.